hack/create-user-openshift.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# This script uses Kubernetes CertificateSigningRequest (CSR) to a generate a
+# certificate signed by the Kubernetes CA itself.
+# It requires cluster admin permission.
+#
+# e.g.: ./create-user.sh alice oil
```

Comment:
Command name should be `create-user-openshift.sh`

Reply:
Fixed, thank you